importScripts/sql/transformations/LocationHistoryExportView.sql

```diff
@@ -1,133 +1,56 @@
-DROP TABLE IF EXISTS StandardisedLocations;
```

Comment:
This was rudimentary cleaning based on query. The client has now done a full clean of the data so it is not necessary.